### PR TITLE
Fix config loading

### LINKED ISSUE #119: Reenable config loading
Currently config loading ~~and saving~~ is disabled.

### PR DESCRIPTION
**Pull request by anhosi**
Fixes #119

### CODE REVIEW

**Comment by anhosi** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/3812/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/950/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3151/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/3899/)

**Comment by mikaelarguedas** [comment]
@anhosi : FYI the MacOS job was hanging so I killed it and started a new one
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3154/)

File: rviz_common/src/rviz_common/view_manager.cpp [hunk 1]
```diff
@@ -181,7 +181,8 @@ ViewController * ViewManager::getViewAt(int index) const
   if (index < 0) {
     index = 0;
   }
-  return qobject_cast<ViewController *>(impl_->root_property->childAt(index + 1));
+  // TODO(greimela) Figure out why qobject_cast does not work here
+  return dynamic_cast<ViewController *>(impl_->root_property->childAt(index + 1));
```
Comment (wjwwood) [inline rviz_common/src/rviz_common/view_manager.cpp:185]: Perhaps this file needs to be run through AUTOMOC and isn't? Searching around the other reason might be related to us using dlopen to load plugins and then passing objects across that.

**Review by wjwwood — APPROVED** [review]
lgtm

**Pull request merged**